### DISCUSSION TITLE
Improving the election process

### DISCUSSION THREAD

**Discussion by OBWANDO** [o3de/o3de#9585] [General]
Currently as we continue to grow, there are more candidates looking to run for certain positions.
As a result, it can be difficult to determine why one candidate may be better than another or why their election is meaningful.

This is an older template built last year with a series of 6 core questions and an open comment for each candidate to answer so that the community can get to know them better:

---

Why are you running for this position?

What is your vision going forward if you are elected?

What are some of the improvements you see that would be helpful to the project related to this elected position?

What are some of the changes you see that would be helpful the project related to this elected position?

Please provide a list at least 5 of commits or community involvement posts so others may get to know your work better:

Please provide a summary of your background experience (limit it to 4 sentences please):

Please provide any other comments you wish to convey to the community:

---

When an election is started, the SIG will create an open ticket in their specific SIG repo that will serve as the call for candidates for a position with a start and end time in the title. The open ticket should list the position, title, start and end call for candidates, and start/end for voting and review with the announcement date. 

They will then provide the link to the ticket in the Discord, Mailing list, and community media outlets (using marketing if necessary) as a formal call for candidates.

Candidates will then submit a comment into that ticket with the questions above answered for voting and review.
When the voting period comes up, all community members can review and decide on their candidates from the information provided.

---

I would like to have the community weigh in on this discussion and provide more thoughts as well as help refine this process so that as we grow, we can make better decisions for the benefit of us all.

**Comment by lmbr-pip** [comment]
Is the plan to add this to community repro as a guide at some point? I don't see anything here (or at least easily discoverable): https://github.com/o3de/community/ that covers election process, frequency nor requirements (aside from TSC elections)
